api/rs/slint/android.rs

```diff
@@ -33,7 +33,7 @@
 //! }
 //! ```
 //!
-//! That function must be in a `cdylib` library, and you should enable the "backend-android-activity-06"`
+//! That function must be in a `cdylib` library, and you should enable the `backend-android-activity-06"`
```

Comment:
I think the trailing `\`` is the issue here, not the starting `"`.

Reply:
Yes, I made a mistake..

Modified. Please review again.